protobuf/riak-protobuf.cabal

```diff
@@ -15,122 +15,29 @@ copyright:           Copyright 2011 MailRank, Inc.
 category:            Network
 build-type:          Simple
 extra-source-files:
-  README.markdown
-  proto.sh
-  src/riak.proto
-  src/riak_kv.proto
-  src/riak_search.proto
-  src/riak_dt.proto
-  src/riakextra.proto
   Changes.md
+  README.markdown
+  generate-proto.sh
+  proto/riak.proto
```

Comment:
How much do we value matching the file names from the *official* source? https://github.com/bet365/riak_pb/tree/develop/src

Not that they seem to have changed in the last 3+ years.

Reply:
Per @llelf I will be reverting this to use multiple files